society-of-biblical-literature-fullnote-bibliography.csl

```diff
@@ -656,42 +828,118 @@
       <text variable="URL"/>
     </group>
   </macro>
-  <citation et-al-min="4" et-al-use-first="1" disambiguate-add-names="true">
-    <layout suffix="." delimiter="; ">
-      <choose>
-        <if position="ibid-with-locator">
-          <group delimiter=", ">
-            <text term="ibid"/>
-            <text macro="point-locators-subsequent"/>
-          </group>
-        </if>
-        <else-if position="ibid">
-          <text term="ibid"/>
-        </else-if>
-        <else-if position="subsequent">
+  <macro name="unsigned-dictionary-note">
+    <group delimiter=", ">
+      <group>
+        <text variable="container-title" font-style="italic" text-case="uppercase"/>
+        <number variable="volume" prefix=" "/>
+        <text variable="collection-title" text-case="uppercase"/>
+        <text variable="collection-number" prefix=" "/>
+      </group>
+      <group delimiter=" ">
+        <text term="sub verbo" form="short"/>
+        <text variable="title" text-case="title" quotes="true"/>
+    </group>
+  </group>
+  </macro>
+  <macro name="signed-dictionary-note">
+    <choose>
+      <if position="subsequent">
           <group delimiter=", ">
             <text macro="contributors-short"/>
-            <text macro="title-short"/>
-            <text macro="disambiguate-chapter"/>
-            <text macro="point-locators-subsequent"/>
-          </group>
-        </else-if>
-        <else>
+            <choose>
+              <if position="ibid" match="none">
+                <!-- Excludes Title for ibid entries per CMS 14.34.-->
```

Comment:
Why do we thing SBL is following CMS 17th edition on this? I'm worried this will cause a lot of confusion.

Reply:
Good point! I'm basing this off guidance from my seminary professors at Bethlehem College and Seminary (mostly Andy Naselli). I think it is permitted currently with SBL, but I don't know that it has been officially adopted. I'll double check and see if we have any documentation confirming they are adopting the style. If not, I can split this preference off and make a non-ibid style to share with my seminary classmates while not including this change in the official pull request.